pkg/disk/limactl_disk.go

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package disk
+
+type limactlDiskListOutput struct {
```

Comment:
Maybe we can add a comment with some reasoning why we can't import this from [lima-vm/pkg/store/disk](https://github.com/lima-vm/lima/blob/97ac8caa9a0b5e7b3d664d5e1c73b0e0b710ac75/pkg/store/disk.go#L14-L21)

Reply:
good point, I actually overlooked this option. I think importing it from the original lima struct is better since we won't need to update it each time a field is added upstream in lima